packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -645,9 +621,6 @@ describe('TokenListController', () => {
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
     controller.start();
     await new Promise<void>((resolve) => setTimeout(() => resolve(), 150));
-    expect(controller.state.tokenList).toStrictEqual(
```

Comment:
Should we update the test name ? as it's referencing a deleted state property `tokenList`